datacube/api/core.py

```diff
@@ -326,36 +356,39 @@ def load(self, product=None, measurements=None, output_crs=None, resolution=None
                         resampling='cubic'
                 )
 
+            odc-geo style xy objects are preferred for passing in resolution and align pairs to avoid x/y ordering
```

Comment:
I'm not sure this is true - in the standard case (e.g. square pixels), I think passing in a single float/integer resolution is what we should recommend (much easier than having to use `odc-geo` XY objects. In my understanding they're only required for non-square pixels (a much rarer use case).

Reply:
(assuming that `odc-geo`'s support for single float/integer resolutions is supported by datacube 1.9)

Reply:
Yeah, I think the general wording here is fine given the specific clarification below.